scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/ComboBoxStringConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import javafx.util.StringConverter;
+
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+
+public class ComboBoxStringConverter extends StringConverter<LightingMode> {
```

Comment:
This is great. We just need to add a JavaDoc style comment, to explain a little bit what this class does:

```suggestion
/**
 * Converts the LightningMode values to strings to display the selected value in the ComboBox.
 */
public class ComboBoxStringConverter extends StringConverter<LightingMode> {
```